src/common/tracing.ts

```diff
@@ -1,12 +1,35 @@
 import { Tracing } from '@map-colonies/telemetry';
-import { ExpressInstrumentation } from '@opentelemetry/instrumentation-express';
+import { SpanOptions } from '@opentelemetry/api';
+import * as api from '@opentelemetry/api';
+import { AsyncHooksContextManager } from '@opentelemetry/context-async-hooks';
 import { HttpInstrumentation } from '@opentelemetry/instrumentation-http';
-import { IGNORED_INCOMING_TRACE_ROUTES, IGNORED_OUTGOING_TRACE_ROUTES } from './constants';
+import { ITraceParentContext } from './interfaces';
+import { SERVICE_NAME, SERVICE_VERSION } from './constants';
 
-export const tracing = new Tracing([
-  new HttpInstrumentation({
-    ignoreIncomingPaths: IGNORED_INCOMING_TRACE_ROUTES,
-    ignoreOutgoingUrls: IGNORED_OUTGOING_TRACE_ROUTES,
-  }),
-  new ExpressInstrumentation(),
-]);
+const contextManager = new AsyncHooksContextManager();
+contextManager.enable();
+api.context.setGlobalContextManager(contextManager);
+
+export const tracing = new Tracing(
+  [new HttpInstrumentation({ requireParentforOutgoingSpans: true })],
+  {
+    // eslint-disable-next-line @typescript-eslint/naming-convention
+    '@opentelemetry/instrumentation-express': { enabled: false },
```

Comment:
why?

Reply:
the new configuration for optimal tracing according OFER, the other services use deprecated notations